src/nvim/api/vim.c

```diff
@@ -85,6 +86,54 @@ void api_vim_free_all_mem(void)
   map_free(String, handle_T)(namespace_ids);
 }
 
+void nvim_source(String command, Error *err) FUNC_API_SINCE(7)
+{
+  try_start();
+  if (do_source_str(command.data, "nvim_source(..)") == FAIL) {
+    EMSG(_("Error in executing!"));
```

Comment:
This error message is never shown (and shouldn't be needed either, the actual error will already have displayed a message).

Reply:
I have removed the error message.